src/styles/saas-list.less

```diff
@@ -1,76 +1,104 @@
+.build-applications-view {
+
+  h1 {
+    color: @saas-text;
+    margin: 0;
+    padding: 20px;
+    padding-left: (@grid-gutter-width / 2);
+    width: 100%;
+  }
+}
+
 .saas-offerings-container {
-  background: @color-pf-black-200;
-  padding-left: 0px;
 
   .spinner-container {
     padding-top: 60px;
     padding-bottom: 60px;
   }
 
-  .card-view-pf {
-    padding-left: 21px;
-  }
-
-  .card-view-pf .card-content {
-    background-color: @color-pf-white;
-    cursor: pointer;
-    margin: 0;
+  .saas-list {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin: 0 (@grid-gutter-width / 2);
 
     @media (min-width: @screen-sm-min) {
-      margin: 2px 0 10px;
+      flex-direction: row;
     }
-  }
 
-  .card-view-pf .card {
-    height: 100%;
-    width: 100%;
-    cursor: pointer;
-    text-align: inherit;
-    margin-bottom: 8px;
+    .card {
+      align-items: center;
+      border-bottom: solid 1px @saas-text;
+      color: @saas-text;
+      cursor: pointer;
+      display: flex;
+      height: 5em;
+      width: 100%;
 
-    @media (min-width: @screen-sm-min) {
-      height: 200px;
-      width: 260px;
-      margin-bottom: 20px;
+
+      &:hover {
+        background-color: @saas-hover;
+      }
+
+      @media (min-width: @screen-sm-min) {
+        border-bottom: none;
+        flex: 0 1 auto;
+        height: 300px;
+        padding: 30px;
+        width: 280px;
+      }
     }
-  }
 
-  .card-container {
-    padding: 6px;
+    .last {
+      border-bottom: none;
+    }
 
-    .card-header {
-      display: table;
+    .card-content {
+      align-items: center;
+      cursor: pointer;
+      display: flex;
 
       @media (min-width: @screen-sm-min) {
-        height: 70px;
+        display: block;
+        justify-content: space-around;
+        text-align: center;
       }
 
       .card-icon {
-        display: table-cell;
-        vertical-align: middle;
+        font-size: 24px;
         text-align: center;
         width: 40px;
-        font-size: 24px;
 
         &.fa.fa-mobile {
           font-size: 32px;
         }
+
+        @media (min-width: @screen-sm-min) {
+          font-size: 108px;
+          margin-bottom: 20px;
+          width: 100%;
+
+          &.fa.fa-mobile {
+            font-size: 108px;
+          }
+        }
       }
+
       .card-title {
-        vertical-align: middle;
-        display: table-cell;
-        padding-left: 8px;
+        font-size: @font-size-base + 4;
```

Comment:
Perhaps this should be scaled down for resolutions of maybe 480 and less so the text isn't so crowded?

![screen shot 2017-03-31 at 3 24 17 pm](https://cloud.githubusercontent.com/assets/895728/24566161/6ecd35aa-1626-11e7-9293-74d8dd3a4046.PNG)